8/onbuild/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.16.0-stretch
+FROM node:8.16.0-buster
```

Comment:
Drop this change please. This PR should not be changing the default variant.

Also the onbuild variant is deprecated, so this change has little value anyway.